tests/yam/agama/boot_encrypted.pm

```diff
@@ -0,0 +1,44 @@
+## Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Boot the system for encryption scenario.
```

Comment:
"Boot the system with encrypted partitions" and you can remove the text below (which still have those weird `{, }` btw).

Reply:
updated